python-sdk/pyproject.toml

```diff
@@ -107,7 +111,8 @@ all = [
     "protobuf<=3.20", # Google bigquery client require protobuf <= 3.20.0. We can remove the limitation when this limitation is removed
     "openlineage-airflow>=0.17.0",
     "apache-airflow-providers-microsoft-azure",
-    "azure-storage-blob"
+    "azure-storage-blob",
+    "apache-airflow-providers-microsoft-mssql"
```

Comment:
```suggestion
    "apache-airflow-providers-microsoft-mssql>=3.2"
```

Reply:
We should have the same deps in both places

Reply:
yes, made the changes